server/src/main/java/com/linecorp/centraldogma/server/internal/storage/project/DefaultProjectManager.java

```diff
@@ -41,12 +41,13 @@
     @Nullable
     private final RepositoryCache cache;
 
-    public DefaultProjectManager(File rootDir, Executor repositoryWorker, MeterRegistry meterRegistry,
-                                 @Nullable String cacheSpec) {
-        super(rootDir, Project.class);
+    public DefaultProjectManager(File rootDir, Executor repositoryWorker, Executor purgeWorker,
+                                 MeterRegistry meterRegistry, @Nullable String cacheSpec) {
+        super(rootDir, Project.class, purgeWorker);
 
         requireNonNull(meterRegistry, "meterRegistry");
         requireNonNull(repositoryWorker, "repositoryWorker");
+        requireNonNull(purgeWorker, "purgeWorker");
```

Comment:
This is redundant because the super constructor will check nullness.

Reply:
Oops, thanks. You have an eagle eye!  👀